clusters/sqnc-staging/base/flux-system/gotk-sync.yaml

```
@@ -7,7 +7,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: feature/opentelemetry-staging
```

This will need committing when ready to merge:
```suggestion
    branch: main
```